subsys/bluetooth/controller/Kconfig

```diff
@@ -72,6 +72,9 @@ config BT_CTLR_SMI_SUPPORT
 config BT_CTLR_CONN_RSSI_SUPPORT
 	bool
 
+config BT_CTLR_ECDH_SUPPORT
```

Comment:
There should be a similar configuration `BT_CTLR_ECDH` which will allow to application to enable/disable this functionality in the controller. See `BT_CTLR_LE_ENC_SUPPORT` and `BT_CTLR_LE_ENC`

Reply:
Added.